exovetter/lightkurve_utils.py

```diff
@@ -0,0 +1,19 @@
+"""Functions to deal with the changing lightkurve versions"""
+
+import astropy
+
+
+def unpack_lk_version(lightcurve, flux_name):
+    """Code to get time, flux and time_format from lightcurve object
+    independent of v1 or v2 of the package.
+    """
+    if isinstance(lightcurve.time, astropy.time.core.Time):
```

Comment:
I guess this works. Another way is to compare the versions of lightkurve being used using `packaging.version.Version`.